website/src/lib/oasst_inference_client.ts

```diff
@@ -81,6 +75,13 @@ export class OasstInferenceClient {
 }
 
 export const createInferenceClient = (jwt: JWT) => {
-  const token = jwt.inferenceTokens?.access_token.access_token;
+  const info: TrustedClient = {
+    // TODO: use api key from env
+    api_key: "0000",
+    client: "website",
+    user_id: jwt.sub,
```

Comment:
No, this is the id of the user in the frontend db.

Reply:
> No, this is the id of the user in the frontend db.
> 
> After thinking about this, using the data backend id would introduce a lot of coupling between the services, we can still aggregate information in the website if needed

Sure it's up to you. But I think having the data backend would allow the inference to communicate the data backend directly if needed. Or maybe we can store both frontend and data backend id. Also after this, I think we can deprecate all other sign methods of the inference (or just disable them for now) and use the frontend auth for other methods.

Reply:
updated to use the backend user id!